metal_small/beast_settings.txt

```diff
@@ -10,78 +10,312 @@ from beast.physicsmodel.stars import stellib
 from beast.physicsmodel.dust import extinction
 from beast.observationmodel.noisemodel import absflux_covmat
 
-# condensed settings just for simulations
-# (see beast-examples/phat_small for beast_settings.txt with extensive comments)
+# -----------------------------------------------------------------
+# User inputs
+# -----------------------------------------------------------------
+# Parameters that are required to make models and to fit the data
+# -----------------------------------------------------------------
+# AC == automatically created
+# indicates where user's input change is NOT necessary/recommended
+# -----------------------------------------------------------------
 
+# project : string
+#   the name of the output results directory
 project = "beast_metal_small"
 
+# name of the survey
+#  used for the creation of the unique name for each source
 surveyname = "METAL"
 
+# filters : list of strings
+#   full filter names in BEAST filter database
 filters = ['HST_WFC3_F225W', 'HST_WFC3_F275W','HST_WFC3_F336W',
            'HST_ACS_WFC_F475W','HST_ACS_WFC_F814W',
            'HST_WFC3_F110W','HST_WFC3_F160W']
 
+# basefilters : list of strings
+#   short names for filters
 basefilters = ["F225W", "F275W", "F336W", "F475W", "F814W", "F110W", "F160W"]
+
+# obs_colnames : list of strings
+#   names of columns for filters in the observed catalog
+#   need to match column names in the observed catalog,
+#   input data MUST be in fluxes, NOT in magnitudes
+#   fluxes MUST be in normalized Vega units
 obs_colnames = [f.upper() + "_RATE" for f in basefilters]
 
-# not needed?
+# obsfile : string
+#   pathname of the observed catalog
 obsfile = "data/14675_LMC-13361nw-11112.gst_samp.fits"
 
-## artificial star tests
+#----------------------------------
+# Source Density Binning Parameters
+#----------------------------------
```

Comment:
I think this section should be moved to right before the AST parameters.  Just thinking of the flow and this is related to the AST and observation model parameters.  Not needed for the physical model.  

This would also support other uses of the BEAST - like just generating fluxes of stars in specific filters.

Reply:
done!